libs/plpgsql.json

```diff
@@ -0,0 +1,46 @@
+[
+  {"id": "plpgsql.adminpack", "imports": ["adminpack"], "name": "adminpack", "repo": "postgres/postgres/tree/master/contrib/adminpack", "tags": ["plpgsql-admin"], "tech": ["app-blocks"]},
+  {"id": "plpgsql.amcheck", "imports": ["amcheck"], "name": "amcheck", "repo": "postgres/postgres/tree/master/contrib/amcheck", "tags": ["db-indexes-integrity"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.auth-delay", "imports": ["auth_delay"], "name": "auth_delay", "repo": "postgres/postgres/tree/master/contrib/auth_delay", "tags": ["auth-attack-protection"], "tech": ["security"]},
+  {"id": "plpgsql.auto-explain", "imports": ["auto_explain"], "name": "auto_explain", "repo": "postgres/postgres/tree/master/contrib/auto_explain", "tags": ["db-explain"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.bloom", "imports": ["bloom"], "name": "bloom", "repo": "postgres/postgres/tree/master/contrib/bloom", "tags": ["bloom-filters"], "tech": ["utils"]},
+  {"id": "plpgsql.btree-gin", "imports": ["btree_gin"], "name": "btree_gin", "repo": "postgres/postgres/tree/master/contrib/btree_gin", "tags": ["b-tree"], "tech": ["data-structures"]},
+  {"id": "plpgsql.btree-gist", "imports": ["btree_gist"], "name": "btree_gist", "repo": "postgres/postgres/tree/master/contrib/btree_gist", "tags": ["b-tree"], "tech": ["data-structures"]},
+  {"id": "plpgsql.chkpass", "imports": ["chkpass"], "name": "chkpass", "repo": "postgres/postgres/tree/master/contrib/chkpass", "tags": ["store-passwords"], "tech": ["security"]},
+  {"id": "plpgsql.citext", "imports": ["citext"], "name": "citext", "repo": "postgres/postgres/tree/master/contrib/citext", "tags": ["string"], "tech": ["utils"]},
+  {"id": "plpgsql.cube", "imports": ["cube"], "name": "cube", "repo": "postgres/postgres/tree/master/contrib/cube", "tags": ["geometry"], "tech": ["computational-science"]},
+  {"id": "plpgsql.dblink", "imports": ["dblink"], "name": "dblink", "repo": "postgres/postgres/tree/master/contrib/dblink", "tags": ["db-link"], "tech": ["app-arch"]},
+  {"id": "plpgsql.dict-int", "imports": ["dict_int"], "name": "dict_int", "repo": "postgres/postgres/tree/master/contrib/dict_int", "tags": ["text-search"], "tech": ["computational-science"]},
+  {"id": "plpgsql.dict-xsyn", "imports": ["dict_xsyn"], "name": "dict_xsyn", "repo": "postgres/postgres/tree/master/contrib/dict_xsyn", "tags": ["text-search"], "tech": ["computational-science"]},
+  {"id": "plpgsql.earthdistance", "imports": ["earthdistance"], "name": "earthdistance", "repo": "postgres/postgres/tree/master/contrib/earthdistance", "tags": ["geo-coords"], "tech": ["utils"]},
+  {"id": "plpgsql.file-fdw", "imports": ["file_fdw"], "name": "file_fdw", "repo": "postgres/postgres/tree/master/contrib/file_fdw", "tags": ["remote-control"], "tech": ["utils"]},
```

Comment:
Hm... how did you arrive with 'remove-control'? it seems to be just a module to access files. file-access maybe?